src/main/java/rs/hybridit/model/User.java

```diff
@@ -57,4 +75,41 @@ public User(UserDto userDto) {
 
 	}
 
+	public void setAuthority(Role role){
+		List<Authority> authorities = new ArrayList<>();
+		Authority a = new Authority();
+		a.setName(role);
+		authorities.add(a);
+		this.setAuthorities(authorities);
+	}
+
+	@Override
+	public Collection<? extends GrantedAuthority> getAuthorities() {
+		return null;
+	}
+
+	@JsonIgnore
+	@Override
+	public boolean isAccountNonExpired() {
+		return false;
+	}
+
+	@JsonIgnore
+	@Override
+	public boolean isAccountNonLocked() {
+		return false;
```

Comment:
locked..?

Reply:
User implements UserDetails interface so I had to override that method. I am not going to use it, so I just put false there, but yes it is more logical to put true as accounts in my application are never locked.

Reply:
Changed it to true.